run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+LD_LIBRARY_PATH=/app/lib /app/interstellar/interstellar "$@"
```

Comment:
```suggestion
exec /app/interstellar/interstellar "$@"
```

Reply:
This is fixed now, once the "ugly hack" was removed.